Lesson 2 - Functions/code/homework/code.js

```diff
@@ -18,30 +18,74 @@
     global.UAM.aircrafts[0].services.push({
         name: 'smth1',
         timeToExecute: 120
-    });
+    });*/
     
     //////////////////////////////////////////////////////////////////////////////////////
 
     global.UAM.addAircraft = function (newAircraftCode) {
         // function should return new aircraft object
+	
+	global.UAM.aircrafts.push({
+        code: newAircraftCode ,
+        services: []
+    	});
+    
+  	return global.UAM.aircrafts[global.UAM.aircrafts.length-1];
     };
 
     global.UAM.removeAircraft = function (aircraftObj) {
         // !!!
+
+	var i = global.UAM.aircrafts.indexOf(aircraftObj);
+	if(i>-1)
+		global.UAM.aircrafts.splice(i,1);
     };
 
     global.UAM.addWorkToAircraft = function(aircraftObj, name, timeToExxecute) {
         // !!!
+
+	var i = global.UAM.aircrafts.indexOf(aircraftObj);
+	global.UAM.aircrafts[i].services.push({
+        name: name,
+        timeToExecute: timeToExxecute
+    });
+
     };
         
-    global.UAM.reduceTimeToExecute = function(time) {
+    global.UAM.reduceTimeToExecute = function(obj , time) {
         // !!!
+
+		if(obj.services!=null)
+		for(var j =0; j<obj.services.length; j++)
+			{		
```

Comment:
Its my code , I just use it. I know i may use just `i` like in previous `for` statement. Sorry for my english...